fetch_resources.sh

```diff
@@ -538,12 +538,30 @@ function configure_tempest
     done
     TVAULT_IP+="]"
 
+
+    #check for test user name in TEST_IMAGE_NAME
+    #keep field empty in-case image name is other than ubuntu, centos, cirros
+    case $TEST_IMAGE_NAME in
+        *"ubuntu"*)
```

Comment:
Please handle upper and lower case names as well.

Reply:
Updated code for lower case and upper case.